### PR TITLE
apache_conf resource: Strip quotes from values

### PR DESCRIPTION
**Pull request by jerryaldrichiii**
This removes quotes from values captured by SimpleConfig.

Example:

`['\"/etc/apache2\"']`

becomes

`['/etc/apache2']`

This solves the bug portion of #3123

### CODE REVIEW

**Comment by aaronlippold** [comment]
👍 thanks for the quick turn around on this?

**Comment by jquick** [comment]
Would we ever have legitimate spaces in the quotes for any parameters?

**Comment by aaronlippold** [comment]
Not that I recall but I am not a totally an apache conf expert

**Comment by jerryaldrichiii** [comment]
@jquick has a point though. I'll update the regex to capture any chars between quotes (even escaped quotes).

**Review by miah — APPROVED** [review]

File: lib/resources/apache_conf.rb [hunk 1]
```diff
@@ -85,6 +85,12 @@ def read_content
           assignment_regex: /^\s*(\S+)\s+((?=.*\s+$).*?|.*)\s*$/,
           multiple_values: true,
         ).params
+
+        # Capture any characters between quotes that are not escaped in values
+        params.values.map! do |x|
```
Comment (clintoncwolfe) [inline lib/resources/apache_conf.rb]: Could we not use `x` and `y` here?

**Review by clintoncwolfe — CHANGES_REQUESTED** [review]
Keyboards now come with **multiple** letters!

File: lib/resources/apache_conf.rb (hunk 1 above)
Comment (jerryaldrichiii) [reply lib/resources/apache_conf.rb]: Sure can, I changed it to `value` and `sub_value`.

Let me know if you have better names...I'm bad with naming things.

**Review by clintoncwolfe — APPROVED** [review]
Thanks @jerryaldrichiii

**Pull request merged**